src/calendar_duration.rs

```diff
@@ -35,6 +36,19 @@ pub struct CalendarDuration {
     nanos: NonZeroU32,
 }
 
+impl fmt::Debug for CalendarDuration {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        let (mins, secs) = self.mins_and_secs_component();
+        f.debug_struct("CalendarDuration")
```

Comment:
How about we avoid enconding fields that are 0?

Reply:
Isn't it strange for a struct to leave out fields that are 0 from its `Debug` format?

Reply:
Seems fine to me.

Reply:
Especially since the fields are private.

Reply:
Done. It feels strange to me tbh.